server/routes/github/github_repo_selection.py

```
@@ -262,6 +271,69 @@ def update_repo_metadata(user_id, repo_full_name):
+                    # installations can't deliver webhooks, so enabling
+                    # would be a silent no-op — reject those too.
+                    cur.execute(
+                        f"""SELECT r.installation_id, i.suspended_at
+                             FROM connected_repos r
+                             LEFT JOIN github_installations i
+                                    ON i.installation_id = r.installation_id
+                            WHERE r.provider = 'github'
+                              AND r.repo_full_name = %s AND {predicate}
+                            ORDER BY (r.installation_id IS NULL) ASC,
+                                     r.updated_at DESC
+                            LIMIT 1""",
```

Ruff (0.15.15)

[error] Possible SQL injection vector through string-based query construction

(S608)